### PR TITLE
test: update the ci/cd pipeline

### PR DESCRIPTION
**Pull request by chalmerlowe**
Removing the unit tests from the noxfile run initiated by kokoro in deference to the unit tests performed via Github Actions.

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -98,9 +98,7 @@ def wrapper(*args, **kwargs):
 
 # 'docfx' is excluded since it only needs to run in 'docs-presubmit'
 nox.options.sessions = [
-    "unit",
     "system",
```
Comment (chalmerlowe) [inline noxfile.py:101]:
```suggestion
```

**Comment by chalmerlowe** [comment]
@Linchin Can you approve this, please?

**Comment by chalmerlowe** [comment]
@Linchin
Can you review and approve, please?

**Comment by Linchin** [comment]
BLOCKER:
I think owlbot reverted some change. As this is updating the testing pipeline, could you take a look and double check if we should update owlbot accordingly too?

**Review by Linchin — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**